packages/dd-trace/test/plugins/agent.js

```diff
@@ -75,6 +75,18 @@ module.exports = {
     })
   },
 
+  reload (pluginName, config) {
+    pluginName = [].concat(pluginName)
+    plugins = pluginName
+    config = [].concat(config)
+
+    for (let i = 0, l = pluginName.length; i < l; i++) {
+      tracer.use(pluginName[i], config[i])
+    }
+
+    return Promise.resolve()
```

Comment:
Why does this need a promise?

Reply:
It doesn't need to be, I originally implemented it directly in `load` then moved it so it's mostly a leftover. I can remove it.